monkestation/code/game/objects/items/ancient_artifacts.dm

```diff
@@ -0,0 +1,455 @@
+				return
+			damage = target.getToxLoss()
+			target.adjustToxLoss(-damage)
+		if(OXY)
+			if(HAS_TRAIT(user,TRAIT_NOBREATH))
+				to_chat(user,"<span class='warning'>You can't take this damage type!</span>")
+				return
+			damage = target.getOxyLoss()
+			target.adjustOxyLoss(-damage)
+		else
+			to_chat(user,"<span class='warning'>You must select a damage type first!</span>")
+			return
+
+	user.apply_damage(damage,damage_type)
+	if(damage >= 1)
+		user.visible_message("<span class='notice'>[target]'s wounds disappear, as [user] looks more damaged.</span>", \
+							"<span class='notice'>[target] starts to look better, while you feel a little more hurt.</span>")
+	else
+		to_chat(user,"<span class='notice'>Nothing happened, [target] must not've been hurt.</span>")
+
+//Organics Smuggling Bag
+/obj/item/organics_smuggling_bag
+	name = "organics smuggling bag"
+	desc = "A bag made with bluespace tech scientists made while on tons of space drugs. Can fit a lot of people inside."
+	slot_flags = ITEM_SLOT_BACK //Its totally a normal backpack :)
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "backpack"
+	item_state = "backpack"
+
+/obj/item/organics_smuggling_bag/attack_self(mob/living/carbon/user)
+	if(contents.len >= 1)
+		user.visible_message("<span class='warning'>Everyone inside \the [src] tumbles out!</span>")
+		var/turf/tumble_tile = get_turf(user)
+		for(var/mob/living/inhabitant in contents)
+			inhabitant.forceMove(tumble_tile)
+			inhabitant.Knockdown(3 SECONDS)
+	else
+		to_chat(user,"<span class='warning'>There's no one inside!</span>")
+
+/obj/item/organics_smuggling_bag/attack(mob/living/target, mob/living/user)
+	if(!isliving(target))
+		return
+	if(target == user)
+		to_chat(user, "<span class='notice'>You can't get yourself inside!</span>")
+		return
+	if(contents.len > 10)
+		to_chat(user, "<span class='warning'>There are too many people inside!</span>")
+		return
+
+	target.visible_message("<span class='warning'>[user] is trying to stuff [target]\s body into \the [src]!</span>", \
+							"<span class='danger'>[user] is trying to stuff you into \the [src]!</span>")
+	if(do_mob(user, target, 15 SECONDS))
+		target.forceMove(src)
+		user.visible_message("<span class='warning'>[user] put [target] inside \the [src]!</span>")
+	..()
+
+/obj/item/organics_smuggling_bag/examine(mob/user)
+	. = ..()
+	var/number_of_people = contents.len
+	if(number_of_people >= 1)
+		. +=  "It currently has [number_of_people] [number_of_people == 1 ? "person" : "people"] inside."
+	else
+		. += "There's no one inside."
+
+//Soul Binding Contract
+/obj/item/soul_link_contract
+	name = "soul binding contract"
+	desc = "A contract from Hell itself. If you get a corpse to sign it, it brings them back from beyond the grave and binds your souls together."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "paper_onfire"
+	item_state = "paper"
+
+/obj/item/soul_link_contract/afterattack(mob/living/target, mob/living/user)
```

Comment:
This one needs to check for DNR and also offer a chance for the ghost to say no.